snapUtil.c

```diff
@@ -20,13 +20,12 @@ void
 usage(void)
 {
 	(void) fprintf(stderr, "Usage:\n");
-	(void) fprintf(stderr, "\t%s -l <vol>\n", g_pname);
-	(void) fprintf(stderr, "\t%s -c <snap> <vol>\n", g_pname);
-	(void) fprintf(stderr, "\t%s -n <snap> <newname> <vol>\n", g_pname);
-	(void) fprintf(stderr, "\t%s -d <snap> <vol>\n", g_pname);
-	(void) fprintf(stderr, "\t%s -r <snap> <vol>\n", g_pname);
-	(void) fprintf(stderr, "\t%s -s <snap> <vol> <mntpnt>\n", g_pname);
-	(void) fprintf(stderr, "\t%s -u <snap> <vol>\n", g_pname);
+	(void) fprintf(stderr, "\t%s -l <vol>\t\t\t(List all snapshots)\n", g_pname);
+	(void) fprintf(stderr, "\t%s -c <snap> <vol>\t\t(Create snapshot)\n", g_pname);
+	(void) fprintf(stderr, "\t%s -n <snap> <newname> <vol>\t(Rename snapshot)\n", g_pname);
+	(void) fprintf(stderr, "\t%s -d <snap> <vol>\t\t(Delete snapshot)\n", g_pname);
+	(void) fprintf(stderr, "\t%s -r <snap> <vol>\t\t(Revert to snapshot)\n", g_pname);
+	(void) fprintf(stderr, "\t%s -s <snap> <vol> <mntpnt>\t(Mount snapshot)\n", g_pname);
```

Comment:
I love the added documentation, but I think this code assumes a certain width for tabs. The alignment could look weird on systems with a different tab width.

Reply:
Fixed in 944a3665e95a491380138b0fb77177f804a3a2f1